packages/editor/src/components/post-schedule/label.js

```diff
@@ -48,7 +48,7 @@ export function getFullPostScheduleLabel( dateAttribute ) {
 
 	const timezoneAbbreviation = getTimezoneAbbreviation();
 	const formattedDate = dateI18n(
-		// translators: If using a space between 'g:i' and 'a', use a non-breaking space.
+		// translators: Use a non-breaking space between 'g:i' and 'a' if appropriate.
```

Comment:
As Above

:1234: As this applies in a number of locations, I'll stop leaving as above comments everywhere. Are you able to do a search for `// translators` if it needs to change.